api/src/cli/commands/schema/apply.ts

```diff
@@ -0,0 +1,136 @@
+import chalk from 'chalk';
+import { promises as fs } from 'fs';
+import inquirer from 'inquirer';
+import { load as loadYaml } from 'js-yaml';
+import path from 'path';
+import getDatabase from '../../../database';
+import logger from '../../../logger';
+import { Snapshot } from '../../../types';
+import { getSnapshot } from '../../../utils/get-snapshot';
+import { getSnapshotDiff } from '../../../utils/get-snapshot-diff';
+import { applySnapshot } from '../../../utils/apply-snapshot';
+
+export async function apply(snapshotPath: string, options?: { yes: boolean }): Promise<void> {
+	const filename = path.resolve(process.cwd(), snapshotPath);
+
+	const database = getDatabase();
+
+	let snapshot: Snapshot;
+
+	try {
+		const fileContents = await fs.readFile(filename, 'utf8');
+
+		if (filename.endsWith('.yaml') || filename.endsWith('.yml')) {
+			snapshot = (await loadYaml(fileContents)) as Snapshot;
+		} else {
+			snapshot = JSON.parse(fileContents) as Snapshot;
+		}
+
+		const currentSnapshot = await getSnapshot({ database });
+		const snapshotDiff = getSnapshotDiff(currentSnapshot, snapshot);
+
+		if (
+			snapshotDiff.collections.length === 0 &&
+			snapshotDiff.collections.length === 0 &&
+			snapshotDiff.collections.length === 0
```

Comment:
One has to be sure 😅

Reply:
It would be great if we could configure which we wanted and which we didn't. In my use case I only want collections/fields/relations, and it would be problematic if the other tables were included.

Reply:
Lol whoops